pyproject.toml

```diff
@@ -8,6 +8,7 @@ exclude = '''
   | \.venv
   | _build
   | buck-out
+  | __pycache__
```

Comment:
While you're at it, mind adding .vscode here as well? Just got bit by this yesterday!